protocols/braintrust/overview.md

```diff
@@ -0,0 +1,62 @@
+
+- On-chain voting happens on the Braintrust platform (Voting Portal), but is ungated and visible  to any community member for participation or observation.
+
+**Off-Chain Discussion Environments:** Modeled after Uniswap’s approach to community engagement, these serve as non-binding channels for discussion and debate that are critical to reaching both (off-chain) consensus and (on-chain) approval. Discussion occurs within the Braintrust Discord server. Off-chain voting occurs on the Braintrust Snapshot page.
+
+
+### Voting in Braintrust’s Proposal System 
+
+Areas of decision-making in Braintrust’s governance are substantial in scope and include changes to network fees, the reputation system, technical parameters, governance processes, and token reserve management. 
+
+Changes to the areas noted above are made via an on-chain proposal system, adapted from the Compound fork. Over the long term, the governance of the Braintrust network is expected to increasingly decentralize via community-driven governance as well as on-chain governance activities. Another area of exploration for the Braintrust governance system is quadratic voting, which has proven to be an effective scheme for keeping incentives aligned between users of the network and other token holders.
+
+![](https://i.imgur.com/XzUf2FZ.png)
+
+
+##  Decentralized Dispute Resolution 
+
+Proposals include contract code that can be implemented automatically and without Braintrust’s direct intervention. The voting results are recorded. In the case of a disagreement about invoices and payments, Community-Adjudicated Dispute Resolution gives decision-making power to the community. The Dispute Resolution Program is made up of Jury Members and a Jury Arbitrator. 
+
+Jury Members can be any BTRST token holder.  Should a dispute require a judgment, three Jury Members are randomly selected from the pool to weigh in on the decision. In addition, Jury Members may be called to be eyewitnesses. Eyewitnesses weigh in on subjective statements based on their expertise. For example, a claimant may say something like: “The content writer should have been able to complete more output based on the time he invoiced me.” Three eyewitnesses would be called to make a judgment on that statement as to whether it was reasonable or unreasonable. 
+
+Jury Members serve a one-year term and are self-volunteered. Another crucial member of the Dispute Resolution Program is the Jury Arbitrator. Jury Arbitrators are elected through the off-chain Braintrust governance process and serve a 6 month term. They do not issue rulings but walk claimants and disputants through the process, organize the claims and counterclaims, recruit eyewitnesses, and randomly assign Jury Members to serve for a specific claim. Learn more about Braintrust dispute resolution here. 
+
+## Join Us
+
+Reach out to us on [Discord](https://discord.com/invite/braintrust) or [Telegram](https://t.me/braintrustofficial)
+
+## Additional links
```

Comment:
👀

Reply:
Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w